src/lib.rs

```diff
@@ -768,3 +768,15 @@ pub fn libc_rmdir(p: &str) {
     // with int return value
     let _ = unsafe { libc::rmdir(p.as_ptr() as *const i8) };
 }
+
+/// read and parse an i64 data
+pub fn read_i64_from(mut file: File) -> Result<i64> {
+    let mut string = String::new();
+    match file.read_to_string(&mut string) {
+        Ok(_) => string
+            .trim()
+            .parse()
```

Comment:
I might have added a `parse::<i64>` here for clarity, but if the type inference system finds it, then it's all good.